### PR TITLE
Added documentation

### LINKED ISSUE #25: Add docs to inx-participation

### PR DESCRIPTION
**Pull request by alexsporn**

### CODE REVIEW

File: documentation/docs/api_reference.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+description: Links to the Indexer REST API reference.
```
Comment (muXxer) [inline documentation/docs/api_reference.md]: Copy paste error

File: documentation/docs/api_reference.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- REST API
+- API
+- Reference
+---
+
+# API Reference
+
+You can find specifications of the API exposed by INX-Participation in the links below:
+
+- [HORNET Participation Plugin](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md#public-node-endpoints)
```
Comment (aleksei-korolev) [inline documentation/docs/api_reference.md:18]: There is only one link

File: documentation/docs/welcome.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- IOTA
+- Shimmer
+- Node Software
+- Welcome
+- explanation
+---
+
+# Welcome to INX-Participation
+
+INX-Participation is an extension for nodes to enable on-tangle voting.
```
Comment (aleksei-korolev) [inline documentation/docs/welcome.md:18]: Could you please be more specific? How exactly does it enable the voting? Who is voting?

For example,

"INX-Participation extension handles voting requests from the connected clients and calculates the current voting status for them."

File: documentation/docs/welcome.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+- Shimmer
+- Node Software
+- Welcome
+- explanation
+---
+
+# Welcome to INX-Participation
+
+INX-Participation is an extension for nodes to enable on-tangle voting. 
+The extensions maintains its own database separate from that of the node and provides means to track events and votes.
```
Comment (aleksei-korolev) [inline documentation/docs/welcome.md:19]:
```suggestion
The extensions maintains its own database separate from that of the node and provides means to track events and votes.
```

The database part sounds like a detail of implementation. I would move it to the very end of the paragraph or remove it altogether if it is not actually important to know.

File: documentation/docs/welcome.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+You can find more information on how participation events work in the [HORNET Participation Plugin](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md) specification.
```
Comment (aleksei-korolev) [inline documentation/docs/welcome.md:21]:
```suggestion
You can find more information about participation events in the [HORNET Participation Plugin](https://github.com/iota-community/treasury/blob/main/specifications/hornet-participation-plugin.md) specifications.
```

File: documentation/docs/welcome.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## Setup
+
+The recommended setup is to use the provided [Docker images](https://hub.docker.com/r/iotaledger/inx-participation).
+These images are also used in our [HORNET recommended setup using Docker](http://wiki.iota.org/hornet/develop/how_tos/using_docker).
+
+## Configuration
+
+The participation extension is configured by default to connect to your HORNET instance.
```
Comment (aleksei-korolev) [inline documentation/docs/welcome.md:31]:
```suggestion
The participation extension connects to the running HORNET instance by default.
```

Active voice in general sounds more straightforward than passive voice.

File: documentation/docs/welcome.md (hunk 5 above)
Comment (aleksei-korolev) [inline documentation/docs/welcome.md:21]: Is it HORNET or Hornet? I am seeing both versions across the wiki.

@lucas-tortora

File: documentation/docs/welcome.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+You can find all the configuration options in the [configuration section](configuration.md).
+
+## Dashboard
+
+If you are using the [INX-Dashboard](https://github.com/iotaledger/inx-dashboard) on your node, you can manage events directly from your browser.
+
+## API
+
+The extension exposes a custom set of REST APIs that can be used by wallets and applications to find active, past and upcoming participation events and query event results.
```
Comment (aleksei-korolev) [inline documentation/docs/welcome.md:41]:
```suggestion
The extension exposes a custom set of REST APIs that can be used by wallets and applications to find past, active, and upcoming participation events and query event results.
```

File: documentation/sidebars.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+module.exports = {
+    mySidebar: [
+        {
+            type: 'doc',
+            id: 'welcome',
+        },
+        {
+            type: 'category',
+            label: 'References',
```
Comment (aleksei-korolev) [inline documentation/sidebars.js:20]:
```suggestion
            label: 'Reference',
```

File: documentation/sidebars.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+            items: [
+                {
+                    type: 'doc',
+                    id: 'configuration',
+                    label: 'Configuration',
+                },
+                {
+                    type: 'doc',
+                    id: 'api_reference',
+                    label: 'API Reference',
```
Comment (aleksei-korolev) [inline documentation/sidebars.js:30]: Could you please move the files to the `/reference/` subfolder to mirror the ToC structure?

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: documentation/docs/welcome.md (hunk 5 above)
Comment (lucas-tortora) [reply documentation/docs/welcome.md:21]: Unless it's an acronym, which I don't think is the case, I think it should be Hornet.

**Review by muXxer — APPROVED** [review]

**Comment by alexsporn** [comment]
@aleksei-korolev @lucas-tortora you are free to create a PR with the requested changes.

**Pull request merged**